docs/examples/one_tank.md

```diff
@@ -0,0 +1,219 @@
+The maximum depth, i.e. the number of questions asked on each path, is limited to five.
+The learners class is created and the helper method [`learn_offline`](../reference/flowcean/strategies/offline.md#flowcean.strategies.offline.learn_offline) called to start the training process
+
+```python
+regression_learner = RegressionTree(max_depth=5)
+regression_model = learn_offline(
+    train,
+    regression_learner,
+    inputs,
+    outputs,
+)
+```
+
+The `inputs` and `outputs` variables contain the names of the input and output fields in the `train' dataframe.
+
+Secondly a [multi-layer perceptron](../reference/flowcean/learners/lightning.md) is used to create a model.
+This type of model consists of a set of neurones arranged in layers which are connected with the previous layer.
+The value of each neuron is calculated by weighting and summing up the values of the neurons in the previous layer and applying a non-linear function; in this case a [leaky ReLU function](https://en.wikipedia.org/wiki/Rectifier_(neural_networks)#Leaky_ReLU).
+The result can be read from the last layers neuron values.
+This learners implementation uses the [lightning framework](https://lightning.ai/docs/pytorch/stable/) which is a high-level wrapper around the well known [PyTorch](https://pytorch.org/) library.
+
+```python
+perceptron_learner = LightningLearner(
+    module=MultilayerPerceptron(
+        learning_rate=1e-3,
+        input_size=len(inputs),
+        output_size=len(outputs),
+        hidden_dimensions=[10, 10],
+        ),
+    max_epochs=100,
+)
+perceptron_model = learn_offline(
+    train,
+    perceptron_learner,
+    inputs,
+    outputs,
+)
+```
+
+The final step is to evaluate the obtained models.
+This is done to estimate how well they are able to describe the unknown function as described above.
+Flowcean ships with a couple of different [metrices](../reference/flowcean/metrics/index.md) which can be used for this purpose.
+Depending on the underlaying problem, different metrices can be resonable to apply.
+For this example the [`MeanAbsoluteError`](../reference/flowcean/metrics/index.md#flowcean.metrics.MeanAbsoluteError) and [`MeanSquaredError`](../reference/flowcean/metrics/index.md#flowcean.metrics.MeanSquaredError) error are used.
+These metrics are useful when the output of the learned function is a (more or less) continuous value and the deviation from the actual value is of interest.
+The helper method [`evaluate`](../reference/flowcean/metrics/evaluate.md) allows for easy evaluation of multiple metrices for a learned model.
+
+```python
+regression_report = evaluate(
+    regression_model,
+    test,
+    inputs,
+    outputs,
+    [MeanAbsoluteError(), MeanSquaredError()],
+)
+
+perceptron_report = evaluate(
+    perceptron_model,
+    test,
+    inputs,
+    outputs,
+    [MeanAbsoluteError(), MeanSquaredError()],
+)
+```
+
+For this example, the resulting metrics are about[^2]
```

Comment:
Why referencing the paper here? The numerical results are actually independent on whether they are published or not. Do we actually have to cite the paper with the example. As the paper refers to the current snapshot of the framework, I would rather put it (together with potential future references) on the starting page or some other separate place.